resources/model/OpenStudio.idd

```diff
@@ -21990,11 +21990,315 @@ OS:HeatExchanger:AirToAir:SensibleAndLatent,
       \note Yes means that the heat exchanger will be locked out (off)
       \note when the economizer is operating or high humidity control is active
 
-OS:HeatExchanger:FluidToFluid,
-        \min-fields 15
-   A1, \field Handle
+OS:HeatExchanger:Desiccant:BalancedFlow,
```

Comment:
Ok, reviewed the IDD. All fields are either autosizable or required, references seem to be there. Looks good to me!